modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -230,7 +236,9 @@ With the recommendations, you can edit the workload object to add CPU and memory
 
 [NOTE]
 ====
-There must be operating pods in the project before a VPA can determine recommended resources.
+Before a VPA can determine recommended resources and apply the recommendations to new pods, operating pods must exist and be running in the project. 
+
+To obtain the most accurate recommendations from the VPA, wait several minutes for the pods to run and for the VPA to stabilize.
```

Comment:
This is not true. The "most accurate" recommendation is after the VPA has observed for at least 8 days.

Reply:
Thanks. Updated to "at least 8 days".